--- pnet-0.5.2/libgc/configure.in.orig	Thu Feb  6 02:36:13 2003
+++ pnet-0.5.2/libgc/configure.in	Mon Feb 10 22:38:18 2003
@@ -229,15 +229,17 @@
 esac
 
 case "$host" in
+# Arch specific assembler code is disabled for alpha. See the comment in
+# libgc/include/private/gcconfig.h
  alpha-*-openbsd*)
-    machdep="alpha_mach_dep.lo"
+#    machdep="alpha_mach_dep.lo"
     if test x"${ac_cv_lib_dl_dlopen}" != xyes ; then
        AC_MSG_WARN(OpenBSD/Alpha without dlopen(). Shared library support is disabled)
     fi
     ;;
- alpha*-*-linux*)
-    machdep="alpha_mach_dep.lo"
-    ;;
+# alpha*-*-linux*)
+#    machdep="alpha_mach_dep.lo"
+#    ;;
  i?86-*-solaris2.[[89]]*)
     AC_DEFINE(SOLARIS25_PROC_VDB_BUG_FIXED)
     ;;
--- pnet-0.5.2/libgc/configure.orig	Thu Feb  6 02:36:13 2003
+++ pnet-0.5.2/libgc/configure	Mon Feb 10 22:39:28 2003
@@ -3335,15 +3335,15 @@
 
 case "$host" in
  alpha-*-openbsd*)
-    machdep="alpha_mach_dep.lo"
+#    machdep="alpha_mach_dep.lo"
     if test x"${ac_cv_lib_dl_dlopen}" != xyes ; then
        { echo "$as_me:3340: WARNING: OpenBSD/Alpha without dlopen(). Shared library support is disabled" >&5
 echo "$as_me: WARNING: OpenBSD/Alpha without dlopen(). Shared library support is disabled" >&2;}
     fi
     ;;
- alpha*-*-linux*)
-    machdep="alpha_mach_dep.lo"
-    ;;
+# alpha*-*-linux*)
+#    machdep="alpha_mach_dep.lo"
+#    ;;
  i?86-*-solaris2.[89]*)
     cat >>confdefs.h <<\EOF
 #define SOLARIS25_PROC_VDB_BUG_FIXED 1
