--- Makefile.am~	Sat Jan  3 00:10:14 2004
+++ Makefile.am	Fri Mar 26 21:55:48 2004
@@ -54,29 +54,3 @@
 
 endif
 
-if USING_LIBGC
-
-libgc-all-local:
-	(cd libgc && $(MAKE) $(AM_MAKEFLAGS) all)
-
-libgc-clean-local:
-	(cd libgc && $(MAKE) $(AM_MAKEFLAGS) clean)
-
-libgc-distclean-local:
-	(cd libgc && $(MAKE) $(AM_MAKEFLAGS) distclean)
-
-libgc-install-local:
-	(cd libgc && $(MAKE) $(AM_MAKEFLAGS) install)
-
-libgc-uninstall-local:
-	(cd libgc && $(MAKE) $(AM_MAKEFLAGS) uninstall)
-
-else
-
-libgc-all-local:
-libgc-clean-local:
-libgc-distclean-local:
-libgc-install-local:
-libgc-uninstall-local:
-
-endif
--- configure.in~	Sat Mar 13 01:51:05 2004
+++ configure.in	Fri Mar 26 21:55:31 2004
@@ -483,7 +483,7 @@
 dnl Set up libgc-related variables.
 AC_SUBST(GCLIBS)
 if test x$withgc = xyes ; then
-	GCLIBS='$(top_builddir)/libgc/.libs/libgc.a'
+	GCLIBS='-lgc'
 	AC_DEFINE(HAVE_LIBGC, 1, [Define if you are using libgc])
 else
 	GCLIBS=""
@@ -539,10 +539,8 @@
 	dnl	different depending on whether $srcdir is absolute or relative.
 	if echo $srcdir | grep '^[~/]'; then
 		ffisrcdir=$srcdir/libffi
-		gcsrcdir=$srcdir/libgc
 	else
 		ffisrcdir=../$srcdir/libffi
-		gcsrcdir=../$srcdir/libgc
 	fi
 	if test x$withffi = xyes ; then
 		if test ! -d libffi ; then
@@ -564,26 +562,6 @@
 		cd ..
 	fi
 
-	if test x$withgc = xyes ; then
-		if test ! -d libgc ; then
-			mkdir libgc
-		fi
-		cd libgc
-		${CONFIG_SHELL-/bin/sh} ${gcsrcdir}/configure \
-			--enable-threads=$THREADS --host="${host}" \
-			--build="${build}" --target="${target}" --disable-fast-install \
-			--prefix="${prefix}" \
-			--exec-prefix="${exec_prefix}" --bindir="${bindir}" \
-			--sbindir="${sbindir}" --libexecdir="${libexecdir}" \
-			--datadir="${datadir}" --sysconfdir="${sysconfdir}" \
-			--sharedstatedir="${sharedstatedir}" \
-			--localstatedir="${localstatedir}" --libdir="${libdir}/pnet" \
-			--includedir="${includedir}/pnet" \
-			--oldincludedir="${oldincludedir}/pnet" \
-			--infodir="${infodir}" --mandir="${mandir}" \
-			--srcdir="${gcsrcdir}" || exit 1
-		cd ..
-	fi
 fi
 
 dnl Local variables:
