--- pnet-0.8.0/Makefile.am~	2007-02-06 21:54:52.000000000 +0100
+++ pnet-0.8.0/Makefile.am	2014-12-08 21:45:52.143120825 +0100
@@ -1,9 +1,5 @@
 OPT_SUBDIRS =
 
-if USING_LIBFFI
-OPT_SUBDIRS += libffi
-endif
-
 if USING_LIBGC
 OPT_SUBDIRS += libgc
 endif
--- pnet-0.8.0/configure.in~	2014-12-08 21:44:33.000000000 +0100
+++ pnet-0.8.0/configure.in	2014-12-08 21:46:41.479787135 +0100
@@ -559,7 +559,7 @@
 dnl Set up libffi-related variables.
 AC_SUBST(FFILIBS)
 if test x$withffi = xyes ; then
-	FFILIBS='$(top_builddir)/libffi/.libs/libffi.a'
+	FFILIBS='-lffi'
 	AC_DEFINE(HAVE_LIBFFI, 1, [Define if you are using libffi])
 else
 	FFILIBS=""
@@ -692,10 +692,6 @@
 fi
 AM_CONDITIONAL(USING_LIBGC, test x$withgc = xyes)
 
-if test x$withffi = xyes; then
-	AC_CONFIG_SUBDIRS(libffi)
-fi
-
 if test x$withgc = xyes; then
 	AC_CONFIG_SUBDIRS(libgc)
 fi
