--- pnet-0.6.2/libffi/configure.in~	2003-02-28 05:40:05.000000000 +0100
+++ pnet-0.6.2/libffi/configure.in	2004-01-04 00:36:19.321624735 +0100
@@ -173,7 +173,7 @@
   toolexecdir='$(libdir)/gcc-lib/$(target_alias)'
   toolexeclibdir='$(libdir)'
 fi
-toolexeclibdir=$toolexeclibdir/`$CC -print-multi-os-directory 2>/dev/null`
+# toolexeclibdir=$toolexeclibdir/`$CC -print-multi-os-directory 2>/dev/null`
 AC_SUBST(toolexecdir)
 AC_SUBST(toolexeclibdir)
 
--- pnet-0.6.2/libffi/configure~	2003-02-28 05:40:05.000000000 +0100
+++ pnet-0.6.2/libffi/configure	2004-01-04 00:48:13.321607782 +0100
@@ -3648,7 +3648,7 @@
   toolexecdir='$(libdir)/gcc-lib/$(target_alias)'
   toolexeclibdir='$(libdir)'
 fi
-toolexeclibdir=$toolexeclibdir/`$CC -print-multi-os-directory 2>/dev/null`
+# toolexeclibdir=$toolexeclibdir/`$CC -print-multi-os-directory 2>/dev/null`
 
 
 
