--- pnet-0.8.0/configure.in.orig	2014-12-13 22:35:39.036912567 +0100
+++ pnet-0.8.0/configure.in	2014-12-13 22:52:15.920204065 +0100
@@ -362,9 +362,9 @@
 if test "x$suppress_libm" = "xno" ; then
 	AC_CHECK_LIB(m, sin)
 fi
-AC_CHECK_LIB(dl, dlopen)
-AC_CHECK_LIB(socket, connect)
-AC_CHECK_LIB(nsl, gethostbyname)
+AC_SEARCH_LIBS(dlopen, dl)
+AC_SEARCH_LIBS(connect, socket)
+AC_SEARCH_LIBS(gethostbyname, nsl)
 have_libz=no
 if test "x$suppress_libm" = "xno" ; then
 	AC_CHECK_LIB(z, inflate,
