--- pnet-0.8.0/Makefile.am~	2014-12-08 21:59:37.000000000 +0100
+++ pnet-0.8.0/Makefile.am	2014-12-08 22:07:36.989778046 +0100
@@ -1,9 +1,5 @@
 OPT_SUBDIRS =
 
-if USING_LIBGC
-OPT_SUBDIRS += libgc
-endif
-
 if PNET_TOOLS
 
 OPT_SUBDIRS += dumpasm \
--- pnet-0.8.0/configure.in~	2014-12-08 21:59:37.000000000 +0100
+++ pnet-0.8.0/configure.in	2014-12-08 22:08:33.553110970 +0100
@@ -682,7 +682,7 @@
 dnl Set up libgc-related variables.
 AC_SUBST(GCLIBS)
 if test x$withgc = xyes ; then
-	GCLIBS='$(top_builddir)/libgc/.libs/libgc.a'
+	GCLIBS='-lgc'
 	AC_DEFINE(HAVE_LIBGC, 1, [Define if you are using libgc])
 	if test x$typedalloc = xtrue ; then
 		AC_DEFINE(IL_USE_TYPED_ALLOCATION, 1, [Define if you are using typed allocation with libgc])
@@ -692,10 +692,6 @@
 fi
 AM_CONDITIONAL(USING_LIBGC, test x$withgc = xyes)
 
-if test x$withgc = xyes; then
-	AC_CONFIG_SUBDIRS(libgc)
-fi
-
 AC_OUTPUT([
 Makefile
 include/Makefile
